Index: KAIFreeAudiencesBot/appsettings.Development.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"BotConfiguration\": {\r\n    \"BotApiKey\": \"5241514365:AAEslKGKUQq6LboHjoKnwUY4NvRiU1njmSw\"\r\n  },\r\n  \"ConnectionStrings\": {\r\n    \"ScheduleConnectionSqlite\": \"Filename=Schedule.db;\"\r\n  },\r\n  \"NgrokExecutionPath\" : \"~/Projects/ngrok\",\r\n  \"Logging\": {\r\n    \"LogLevel\": {\r\n      \"Default\": \"Information\",\r\n      \"Microsoft.AspNetCore\": \"Warning\"\r\n    }\r\n  }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/KAIFreeAudiencesBot/appsettings.Development.json b/KAIFreeAudiencesBot/appsettings.Development.json
--- a/KAIFreeAudiencesBot/appsettings.Development.json	(revision 760351544236c0432092a6e4d9d930164ee1b5df)
+++ b/KAIFreeAudiencesBot/appsettings.Development.json	(date 1655120812524)
@@ -5,7 +5,7 @@
   "ConnectionStrings": {
     "ScheduleConnectionSqlite": "Filename=Schedule.db;"
   },
-  "NgrokExecutionPath" : "~/Projects/ngrok",
+  "NgrokExecutionPath" : "C:\\Users\\yoreh\\RiderProjects\\KAIFreeAudiencesBot(Archived)\\ngrok.exe",
   "Logging": {
     "LogLevel": {
       "Default": "Information",
